src/transformers/generation_tf_utils.py

```diff
@@ -1488,9 +1495,7 @@ def _generate(
 
         if self.config.is_encoder_decoder:
             # if model is encoder decoder model, we create encoder_outputs and add to `model_kwargs`
-            model_kwargs = self._prepare_encoder_decoder_kwargs_for_generation(
-                input_ids, return_dict_in_generate, model_kwargs
-            )
+            model_kwargs = self._prepare_encoder_decoder_kwargs_for_generation(input_ids, model_kwargs)
```

Comment:
(nit) you could maybe put the under the `# 4. Prepare ...` comment and change the comment to `prepare model inputs which will be used for ...`